UI/data/locale/en-US.ini

```
@@ -319,7 +319,7 @@ TitleBar.Scenes="Scenes"
 
 # "name already exists" dialog box
 NameExists.Title="Name already exists"
-NameExists.Text="The name is already in use."
+NameExists.Text="This name is already used by another scene or source."
```

I like the existing wording but agree the addition is useful.

```suggestion
NameExists.Text="This name is already in use by another scene or source."
```